api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -749,6 +771,12 @@ private String getUiUrlAsHref() {
     return href(url, url);
   }
 
+  private String getTosUrlAsHref() {
+    final String url =
+        String.format("%s/aou-tos.html", workbenchConfigProvider.get().server.uiBaseUrl);
+    return href(url, "All Of Us Researcher Workbench General Terms of Use");
```

Comment:
Should we give the ability of user to accept the TOS as well?

Reply:
That's an interesting idea.  I'll ask in `#drc-terra-tos`

Reply:
made the "logging in" text a link to the homepage